src/styles/main.scss

```diff
@@ -0,0 +1,51 @@
+div * {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 10px;
+  border-radius: 5px;
+  transform-origin: 0 top;
+}
+
+.container {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  margin: 50px auto;
+  background-color: #e9e9e9;
```

Comment:
It is a good practice to make all colors variable
And move variables to separate file